src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerMaintainLinkCaseTest.java

```diff
@@ -29,6 +36,9 @@ class CaseWorkerMaintainLinkCaseTest {
     @InjectMocks
```

Comment:
Change class name to CaseworkerMaintainLinkCaseTest

Reply:
Done